client/src/modules/reports/generate/configurable_analysis_report/configurable_analysis_report.html

```diff
@@ -0,0 +1,77 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.CONFIGURABLE_ANALYSIS_REPORT.TITLE</h3>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+          <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
+            <bh-fiscal-period-select
+              on-select-fiscal-callback="ReportConfigCtrl.onSelectFiscal(fiscal)"
+              on-select-period-from-callback="ReportConfigCtrl.onSelectPeriodFrom(period)"
+              on-select-period-to-callback="ReportConfigCtrl.onSelectPeriodTo(period)">
+            </bh-fiscal-period-select>
+
+            <hr>
+            <bh-multiple-cashbox-select
```

Comment:
I'm not sure if this is a problem with the form or the component, but the input is not highlighted in red when it is required.